README.md

```diff
@@ -27,3 +27,12 @@ compatibility with each supported tool after every change to the code base.
 Compatibility between software tools and imaging formats is a necessary bridge
 to ensure the reproducibility of results and enable the optimization and evaluation
 of current image processing and analysis workflows.
+
+## Integration with *NiBabel*
+*NiTransforms* started as a feature-repo spun off of *NiBabel*.
+Shortly after starting with [nipy/nibabel#656](https://github.com/nipy/nibabel/pull/656), it became apparent that it was going to build up in a humongous PR nobody would be able to review as thoroughly as it would require.
+Also, *NiTransforms* has many connections to BIDS/BIDS-Derivatives and its X5 format specification for transforms, which falls outside of the current scope of *NiBabel*.
+
+The plan is to make it an isolated tool, and once it is finished, proceed with the integration into *NiBabel*.
```

Comment:
"once it is finished" is pretty ambiguous - perhaps we can brainstorm a checklist

Reply:
```suggestion
The plan is to make it an isolated tool, and once it is pertinent, proceed with the integration into *NiBabel*.
```